packages/shared/lib/core/wallet/actions/sendAmount.ts

```diff
@@ -0,0 +1,13 @@
+import { selectedAccount } from '@core/account'
+import { AddressWithAmount, TransactionReceipt, TransferOptions } from '@iota/wallet'
+import { get } from 'svelte/store'
+
+export function sendAmount(receipientAddress: string, amount: number): Promise<TransactionReceipt[]> {
```

Comment:
Is this function as an own action necessary?  I think we would always use the `trySendAmount` function in the app. If it is like that, I'd place it in the `trySendAmount.ts` without `export`ing it